README.md

```diff
@@ -40,6 +40,12 @@ documentation at https://articlequality.readthedocs.io .
 2. install the package itself and its dependencies `python setup.py install`
 3. You can verify that your installation worked by running `make enwiki_models` to build the English Wikipedia article quality model or `make wikidatawiki_models` to build the item quality model for Wikidata
 
+### Retraining the models
+
+To retrain a model, run `make -B MODEL` i.e `make -B models/wikidatawiki.item_quality.gradient_boosting.model`.
+
+If a model file exists already in the models folder you might get an already up-to-date log.
```

Comment:
I'm not exactly sure what this sentence is referring to. If I run `make -B models/wikidatawiki.item_quality.gradient_boosting.model` multiple times, it seems to retrain the model from scratch every time? At least that was my understanding of what the `-B` flag does.

Reply:
if you don't put the -B it wont retrain the model, because it asumes it is already done, even if it's an old model. So yes, this is the instruction of how to retrain the model.

Reply:
Sounds good. If you touch the files it means you are just altering the timestamp??

Reply:
Yes: https://man7.org/linux/man-pages/man1/touch.1.html

(It also creates an empty file if it doesn't exist yet, but that's not relevant here)